doc/index.rst

```diff
@@ -251,6 +251,29 @@ So either `ABICHECK_URL` or `ABICHECK_FALLBACK` must be set.
 The url should point to an archive (\*.tar.\*,\*.zip, \*.tgz or \*.tbz2). The file's basename will be displayed as version.
 As an alternative `ABICHECK_VERSION` can be provided explicitly.
 
+Example ABI checking
+++++++++++++++++++++
+
+Let's look at a few example configs in `.travis.yml in this repository, industrial_ci <https://github.com/ros-industrial/industrial_ci/blob/master/.travis.yml>`_. Note that all of these examples check the external repositories (i.e. not industrial_ci) specified by `_EXTERNAL_REPO`.
+
+Case-1: Check ABI compatibility between current and the specific version.
+
+```
+- ROS_DISTRO=indigo _EXTERNAL_REPO='ros/actionlib#38ce66e2ae2ec9c19cf12ab22d57a8134a9285be' ROS_REPO=ros ABICHECK=true
```

Comment:
In case of `ros_canopen` this case example this case should read:
```- ROS_DISTRO=kinetic ABICHECK=true ABICHECK_URL=https://github.com/ros-industrial/ros_canopen/archive/0.7.5.tar.gz`
```